internal/code-style-validation.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       packages:
       - PySide2
       - pre-commit
-      - git+https://github.com/shotgunsoftware/tk-toolchain.git@${{ parameters.tk_toolchain_ref }}#egg=tk-toolchain
+      - https://github.com/shotgunsoftware/tk-toolchain/archive/${{ parameters.tk_toolchain_ref }}.zip
```

Comment:
I also discovered another syntax recently but I never tried:
```suggestion
      - tk-toolchain @ https://github.com/shotgunsoftware/tk-toolchain/archive/${{ parameters.tk_toolchain_ref }}.zip
```

https://pip.pypa.io/en/stable/reference/requirements-file-format/

Reply:
Do you know if this is supported by older versions of pip?

Reply:
Not supported in Azure Pipelines

![image](https://github.com/user-attachments/assets/9f0cef50-ae33-4cc9-98cd-d2d59e033477)